contributing/scc.txt

```diff
@@ -71,24 +71,50 @@ are queried at the prompt::
 scc merge
 ---------
 
-.. program:: scc merge
-
 Merge all the PRs based on specified branch matching the input filters
 including all submodules.
 
+Description
+^^^^^^^^^^^
+
+Filters of different types can be specified and combined to include and
+exclude a set of PRs in the merge command. Each filter needs to be formatted
+as `key:value`. If no key but only a value is specified, it is assumed the
+filter is a label filter (see below). These filters can be passed to an :option:`scc merge --include` or :option:`scc merge --exclude` option.
+
+The available filter types are described below:
+
+- Label filters can be specified using the `label` key i.e. `label:<LABEL>`.
+  This filter type will select a Pull Request if one the following condition
+  is met in order of precedence:
+
+  1. a label named `<LABEL>` is applied to the Pull Request
+  2. the Pull Request description contains a line starting with `--<LABEL>`
+  3. one of the Pull Request comments contains a line starting with
+     `--<LABEL>`. Note this comment needs to be written by one of the public
+     members of the organization owning the upstream repository.
+
+- User filters can be specified using the `user` key i.e. `user:<USERNAME>`.
+  This filter type will select a Pull Request if it has been opened by the
+  user `USERNAME`. Additionally, two special user values are allowed:
+
+  1. the `#org` value will filter all PRs opened by public members of the
+     organization of the upstream repository
```

Comment:
as for all filters, it can be used either as an `inclusive` or `exclusive` filter. I agree the choice of `filter` as the verb is suboptimal, I could use `select` or `specify` instead